client/tests/auth.spec.ts

```diff
@@ -0,0 +1,118 @@
+    await page.goto('http://localhost:5173/register');
+    await page.getByPlaceholder('Name').click();
+    await page.getByPlaceholder('Name').fill('test');
+    await page.getByPlaceholder('Name').press('Tab');
+    await page.getByPlaceholder('Email').fill('test@gmail.com');
+    await page.getByPlaceholder('Password').click();
+    await page.getByPlaceholder('Password').fill('gggggggggggggg');
+    await page.getByRole('button', { name: 'Register' }).click();
+    await expect(page.getByText('passwords must be at least 8 characters and contain 1 uppercase, 1 lowercase, 1 number and 1 symbols')).toBeVisible();
+});
+
+test('test week password symbol missing', async ({ page }) => {
+    await page.goto('http://localhost:5173/register');
+    await page.getByPlaceholder('Name').click();
+    await page.getByPlaceholder('Name').fill('test');
+    await page.getByPlaceholder('Name').press('Tab');
+    await page.getByPlaceholder('Email').fill('test@gmail.com');
+    await page.getByPlaceholder('Password').click();
+    await page.getByPlaceholder('Password').fill('Ggggggg3');
+    await page.getByRole('button', { name: 'Register' }).click();
+    await expect(page.getByText('passwords must be at least 8 characters and contain 1 uppercase, 1 lowercase, 1 number and 1 symbols')).toBeVisible();
+});
+
+test('test week password short', async ({ page }) => {
+    await page.goto('http://localhost:5173/register');
+    await page.getByPlaceholder('Name').click();
+    await page.getByPlaceholder('Name').fill('test');
+    await page.getByPlaceholder('Name').press('Tab');
+    await page.getByPlaceholder('Email').fill('test@gmail.com');
+    await page.getByPlaceholder('Password').click();
+    await page.getByPlaceholder('Password').fill('Gg1234!');
+    await page.getByRole('button', { name: 'Register' }).click();
+    await expect(page.getByText('passwords must be at least 8 characters and contain 1 uppercase, 1 lowercase, 1 number and 1 symbols')).toBeVisible();
+});
+
+test('test user already exists', async ({ page }) => {
+    await page.goto('http://localhost:5173/register');
+    await page.getByPlaceholder('Name').click();
+    await page.getByPlaceholder('Name').fill('test');
+    await page.getByPlaceholder('Name').press('Tab');
+    await page.getByPlaceholder('Email').fill('ofek@gmail.com');
+    await page.getByPlaceholder('Password').click();
+    await page.getByPlaceholder('Password').fill('Gg1234!');
+    await page.getByRole('button', { name: 'Register' }).click();
+    await expect(page.getByText('User already exists')).toBeVisible();
+});
+
+test('test new register success', async ({ page }) => {
+    await page.goto('http://localhost:5173/register');
+    await page.getByPlaceholder('Name').click();
+    await page.getByPlaceholder('Name').fill('playwright test');
+    await page.getByPlaceholder('Name').press('Tab');
+    await page.getByPlaceholder('Email').fill(`${Math.random().toString(36).slice(2)}@gmail.com`);
```

Comment:
If you fill the real DB, try to add cleaning after to make sure you don't fill your DB after a while with not real data.